Description: Debian has iceweasel instead of firefox, so this patch changes
 firefox to iceweasel in cinnamon's panel and favorite applications of menu.
Author: Maximiliano Curia <maxy@debian.org>
Origin: Vendor, Debian
Forwarded: not-needed
Last-Update: 2014-06-06
--- a/data/org.cinnamon.gschema.xml.in
+++ b/data/org.cinnamon.gschema.xml.in
@@ -355,7 +355,7 @@
     </key>
 
     <key name="favorite-apps" type="as">
-      <default>[ 'firefox.desktop', 'mintInstall.desktop', 'cinnamon-settings.desktop', 'hexchat.desktop', 'gnome-terminal.desktop', 'nemo.desktop' ]</default>
+      <default>[ 'iceweasel.desktop', 'mintInstall.desktop', 'cinnamon-settings.desktop', 'hexchat.desktop', 'gnome-terminal.desktop', 'nemo.desktop' ]</default>
       <_summary>List of desktop file IDs for favorite applications</_summary>
       <_description>
         The applications corresponding to these identifiers
